lib/hammer_cli_katello/id_resolver.rb

```diff
@@ -141,7 +141,7 @@ def from_lifecycle_environment_id(options)
 
     def search_and_rescue(search_function, resource, options)
       self.send(search_function, scoped_options(resource, options))
-    rescue HammerCLIForeman::MissingSeachOptions # rubocop:disable HandleExceptions
+    rescue HammerCLIForeman::MissingSearchOptions # rubocop:disable HandleExceptions
```

Comment:
Did you mean to update this rather than just test and merge https://github.com/Katello/hammer-cli-katello/pull/391 after this gets merged?